python/python-kafka-consumer.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: python-kafka-consumer
+  name: python-kafka-consumer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: python-kafka-consumer
+  template:
+    metadata:
+      labels:
+        app: python-kafka-consumer
+    spec:
+      containers:
+        - name: python-kafka-consumer
+          image: docker.io/owenerz/python-kafka-consumer:latest
```

Comment:
Don't forget to fix this image path to the strimzi repo in the quay registry

Reply:
Noted